src/d/semantic/caster.d

```diff
@@ -650,3 +681,96 @@ struct Caster(bool isExplicit, alias bailoutOverride = null) {
 		return CastKind.Invalid;
 	}
 }
+
+@("Test implicit casts of floats to/from BasicTypes")
+unittest {
+	import source.location;
+	with (BuiltinType) {
```

Comment:
This doesn't seem to be bringing much, considering the uses are almost entirely references from lists.

Reply:
gone